conversions/decimal_to_binary.py

```diff
@@ -52,7 +52,58 @@ def decimal_to_binary(num: int) -> str:
     return "0b" + "".join(str(e) for e in binary)
 
 
+def decimal_to_binary_recursive(decimal: int) -> str:
+    """
+    Take a positive integer value and return its binary equivalent.
+    >>> decimal_to_binary_recursive(1000)
+    '1111101000'
+    >>> decimal_to_binary_recursive("72")
+    '1001000'
+    >>> decimal_to_binary_recursive("number")
+    Traceback (most recent call last):
+        ...
+    ValueError: invalid literal for int() with base 10: 'number'
+    """
+    decimal = int(decimal)
+    if decimal in (0, 1):  # Exit cases for the recursion
+        return str(decimal)
+    div, mod = divmod(decimal, 2)
+    return decimal_to_binary_recursive(div) + str(mod)
+
+
+def decimal_to_binary_recursive_main(number: str) -> str:
```

Comment:
For simplicity, can we rename the main function `decimal_to_binary_recursive` rather than `decimal_to_binary_recursive_main`? We can then rename the recursive helper function to `decimal_to_binary_recursive_helper` or `decimal_to_binary_recursive_no_prefix` or something like that